### DISCUSSION TITLE
Implementation of 0.10.0

### DISCUSSION THREAD

**Discussion by gitfvb** [Giorgi/DuckDB.NET#170] [General]
Hi @Giorgi 

This project is pretty cool I have to say. Unfortunately I am working with PowerShell and cannot support with C#, but do you think, DuckDB 0.10.0 could be integrated soon into DuckDB.NET or is it a bitter chunk of todos? The new version works pretty nicely with csv files.

Thanks
Florian

**Comment by Giorgi** [comment]
I will release a new version when DuckDB 0.10.1 is released.

**Reply by gitfvb** [reply]
Nice! Thank you very much, happy to test some cases once I can download the dlls.

**Reply by Giorgi** [reply]
FYI, 0.10.1 is planned in the second week of March.

**Reply by gitfvb** [reply]
Ok, that makes sense. Thanks @Giorgi
